### PR TITLE
Tickets/dm 23470

### PR DESCRIPTION
**Pull request by rbovill**

### CODE REVIEW

File: Jenkinsfile.conda [hunk 1]
```diff
@@ -1,8 +1,8 @@
 pipeline {
     agent any
     environment {
-        idl_version = "${GIT_TAG_NAME}"
-        xml_version = "4.6.0"
+        idl_version = sh(returnStdout: true, script: "git tag --sort version:refname | tail -1").trim()
+        xml_version = "v4.6.0"
```
Comment (tribeiro) [inline Jenkinsfile.conda]: I though about a workaround it. What if we add an environment variable in the conda_package_builder with the xml_version? Then, when we update the xml, we update the container with the new tag.

Comment (rbovill) [reply Jenkinsfile.conda]: How about parameterizing the XML version as a build variable?

Comment (tribeiro) [reply Jenkinsfile.conda]: if that works with the git trigger it would be nice

Comment (rbovill) [reply Jenkinsfile.conda]: Work for this is tracked in https://jira.lsstcorp.org/browse/DM-23491

**Pull request merged**